README.md

```diff
@@ -1,8 +1,27 @@
-# GoMint
+<h1 align="center">
+  <br>
+  <a href="https://github.com/GoMint/GoMint">
+  <img src="https://gomint.io/wp-content/uploads/2015/08/cropped-GoMint_Transparent.png" alt="Markdownify" width="200"></a>
+  <br>
+  GoMint
+  <br>
+</h1>
 
-GoMint is a Minecraft PocketEdition server implementation that is still work in progress. The goal is
-to provide a full, modifiable server with plugin support for Java.
+<h4 align="center">GoMint is a Minecraft PocketEdition server implementation that is still work in progress.</h4>
+[![GitHub Stars](https://img.shields.io/github/stars/GoMint/GoMint.svg)](https://github.com/GoMint/GoMint/stargazers) 
+[![GitHub Issues](https://img.shields.io/github/issues/GoMint/GoMint.svg)](https://travis-ci.org/GoMint/GoMint/issues)
+[![Current Version](https://img.shields.io/badge/version-1.1.0-green.svg)](https://github.com/IgorAntun/node-chat) 
```

Comment:
The software in master is version 1.0.5 and the github link is wrong

Reply:
whoops, should be fixed now with the latest commit.